open-craft/xblock-poll
# BB-1975: Update translations

This PR updates translations and make CI checks green again. 

**Testing instructions:**
1. Check that CI checks **related to translations** are green.
2. (optional) Run `make validate_translations` locally.
